### DISCUSSION TITLE
Deleting instances quickly causes it to not fully delete it.

### DISCUSSION THREAD

**Discussion by gwashark** [pockethost/pockethost#438] [Q&A]
# Problem
![image](https://github.com/user-attachments/assets/ecfe8845-6864-47df-8a67-99c6bb818e1a)

# Steps 
1. Turn off Instance.
2. Delete the instance
3. Click ok to the popup

# What to expect:
To go to the dashboard with the instance removed.
## But got:
The instance is still registered but not found in the system.

**Comment by benallfree** [comment]
This is because it doens't create the filesystem entry until first run. I'll take care of it soon.
